lib/Basic/Unix/TaskQueue.inc

```diff
@@ -35,7 +35,10 @@
 #if !defined(__APPLE__)
 extern char **environ;
 #else
-#include <crt_externs.h> // for _NSGetEnviron
+extern "C" {
+    // _NSGetEnviron is from crt_externs.h which is missing in the iOS SDK.
+    extern char ***_NSGetEnviron(void);
```

Comment:
It appears to be fine, it is currently this way in compiler-rt for ASAN. https://github.com/llvm-mirror/compiler-rt/commit/b83cbf0d82572a98f5a3ccbfc460233b93967165